<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtHelpConfigUI</class>
 <widget class="QWidget" name="QtHelpConfigUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>389</height>
   </rect>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Install Additional Documentation Files</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTreeWidget" name="qchTable">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="columnCount">
      <number>5</number>
     </property>
     <property name="showGrid" stdset="0">
      <bool>false</bool>
     </property>
     <attribute name="headerVisible">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string comment="@title:column">Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string comment="@title:column">Path</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">IconName</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">Ghns</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true"/>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="tableCtrlLayout">
     <item>
      <widget class="QPushButton" name="addButton">
       <property name="text">
        <string comment="@action:button">Add</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
